packages/react/scripts/jsx-to-tsx

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+/**
+ * Creates a tsx copy of the passed js/jsx file and tries to convert it into TS.
+ *
+ * Arguments:
+ * - source path of the js/jsx file,
+ * - (optional) `--overwrite` to overwrite the already existing tsx file.
+ */
+
+const { existsSync, statSync, copyFileSync } = require('fs')
+const { exec } = require("child_process");
+
+const args = process.argv.slice(2)
+const sourceFilepath = args[0];
+if (!sourceFilepath) {
+  console.error('Please pass the source file path.');
+  process.exit(1);
```

Comment:
should we let the default to convert all components?

Reply:
Just to keep in text what we discussed already. This script was developed for the manual conversion. Right now it could be used for debugging the TS conversion of a single component.